manifests/reposync-crd.yaml

```diff
@@ -1201,7 +1201,7 @@ spec:
                       CLI. If `values` is also specified, fields from `values` will
                       override fields from valuesFileRefs.
                     items:
-                      description: ValuesFileRefs holds references to ConfigMap objects
+                      description: ValuesFileRef holds references to ConfigMap objects
```

Comment:
why singular? it's talking about the list still, right?

Reply:
This is referring to the singular struct: https://github.com/GoogleContainerTools/kpt-config-sync/blob/547b4aa8f835147d078d2a5e944b903683097a9e/pkg/api/configsync/v1beta1/helmconfig.go#L111

The list field is still plural, so `valuesFileRefs` is now type `[]valuesFileRef`, which seemed like it made more sense.

Reply:
The full CRD description reads like this:

```
valuesFileRefs:
                    description: valuesFileRefs holds references to objects in the
                      cluster that represent values to use instead of default values
                      that accompany the chart. Currently, only ConfigMaps are supported,
                      and the ConfigMap must be in the same namespace as the RootSync
                      or RepoSync. When multiple valuesFiles are used, duplicated
                      keys in later files will override the value from earlier files.
                      This is equivalent to passing in multiple valuesFiles to Helm
                      CLI. If `values` is also specified, fields from `values` will
                      override fields from valuesFileRefs.
                    items:
                      description: ValuesFileRef holds references to ConfigMap objects
                        in the cluster that represent values to use instead of default
                        values that accompany the chart.
                      properties:
                        name:
                          description: name represents the Object name. Required.
                          type: string
                        valuesFile:
                          default: values.yaml
                          description: valuesFile represents the object data key to
                            read the value from.
                          type: string
                      type: object
                    type: array
```

Reply:
if this is a description of a single item then it needs some work...

ValuesFileRef references a ConfigMap object that contains a values file to use for helm rendering. The ConfigMap must be in the same namespace as the RootSync/RepoSync.

Reply:
updated the description